packages/app/public/index.html

```diff
@@ -41,7 +41,7 @@
       href="<%= publicPath %>/safari-pinned-tab.svg"
       color="#5bbad5"
     />
-    <title><%= config.getString('app.title') %></title>
```

Comment:
Should we add this to the changeset especially? Or are we happy with it just being in the upgrade helper?

Reply:
We've effectively stopped repeating the diff in the changeset at this point, so I think it's alright without